### ISSUE TITLE
Grapher step not working: Sources is not of length 1

### ISSUE THREAD

**Issue by lucasrodes** [owid/etl#380]
I am using [yield_long_table](https://github.com/owid/etl/blob/b2baf50c8aec93c07c8e1d44a495573fe6510bc9/etl/grapher_helpers.py#L170) to insert my long table into Grapher.

I am im branch `feature/unwpp`

When I run

```
etl --force --only grapher://un/2022-07-11/un_wpp --grapher
```

I get this error:

```
2022-08-10 19:37.13 [error    ] Error encountered during import: Variable `infant_mortality_rate` must have exactly one source, see function `adapt_table_for_grapher` that can do that for you
```

My table has as index: entity_id, year, sex, age, variant (i.e. I have dimensions)

**Comment by lucasrodes** [comment]
Ideally I'd use [adapt_table_for_grapher](https://github.com/owid/etl/blob/master/etl/grapher_helpers.py#L376), but I believe this should be applied when the data comes in _wide_ format, which is not my case.

**Comment by lucasrodes** [comment]
This has been addressed in #383 -- thanks @larsyencken
